packages/react/src/experimental/SelectPanel2/__tests__/SelectPanelLoading.test.tsx

```diff
@@ -1,22 +1,95 @@
+import {describe, expect, it, vi, beforeEach, afterEach} from 'vitest'
 import {render} from '@testing-library/react'
 import {SelectPanel} from '../'
 
-jest.useFakeTimers()
+vi.useFakeTimers()
+
+function getLiveRegion(): MockLiveRegionElement {
+  const liveRegion = document.querySelector('live-region')
+  if (liveRegion) {
+    return liveRegion as MockLiveRegionElement
+  }
+  throw new Error('No live-region found')
+}
+
+// Mock implementation of the live-region custom element for testing
+class MockLiveRegionElement extends HTMLElement {
```

Comment:
The live region element does not need to be mocked, use the custom element directly